### PR TITLE
Add action set for motion_change_recommendation. Not complete model.

### PR DESCRIPTION
**Pull request by reiterl**
Another action set: motion_change_recommendation, 
Model is not complete, some kind of fields are missing.
Add tests built on meeting tests.

### CODE REVIEW

File: openslides_backend/models/motion_change_recommendation.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    """
+
+    collection = Collection("motion_change_recommendation")
+    verbose_name = "motion_change_recommendation"
+
+    id = fields.IdField(description="The id of this motion change recommendation.")
+    title = fields.RequiredCharField(
+        description="The title of this motion change recommendation."
+    )
+    type = fields.PositiveIntegerField("The type of this motion change recommendation.")
```
Comment (jsangmeister) [inline openslides_backend/models/motion_change_recommendation.py:21]: Add comment:
Replacement (0), Insertion (1), Deletion (2), Other (3)
add max=3 (see #101, you can use any jsonschema constraints on the fields)

File: openslides_backend/models/motion_change_recommendation.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        "The line from of this motion change recommendation."
+    )
+    line_to = fields.PositiveIntegerField(
+        "The line to of this motion change recommendation."
+    )
+    # TODO add the other Foreign Keys.
+    # TODO rejected: boolean;
+    # TODO internal: boolean;
+    # TODO text: HTML;
+    # TODO creation_time: datetime;
```
Comment (jsangmeister) [inline openslides_backend/models/motion_change_recommendation.py:35]: Boolean fields and Timestamp fields (datetime) implemented in #101. HTML field is a text field for now, maybe we'll add a special field there later for validation.

File: openslides_backend/models/motion_change_recommendation.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/models/motion_change_recommendation.py]: Add comment and max=3.

File: openslides_backend/models/motion_change_recommendation.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/models/motion_change_recommendation.py]: Merged #101 and added the fields.

File: openslides_backend/models/motion_change_recommendation.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        "The line to of this motion change recommendation."
+    )
+    rejected = fields.BooleanField(
+        description="If this motion change recommendation is rejected"
+    )
+    internal = fields.BooleanField(
+        description="If this motion change recommendation is internal"
+    )
+    text = fields.TextField(description="The text of the motion change recommendation.")
+    creation_time = fields.TimestampField(
```
Comment (jsangmeister) [inline openslides_backend/models/motion_change_recommendation.py:38]: The creation time should be set to the current timestamp on create.

File: openslides_backend/models/motion_change_recommendation.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    Reverse fields:
+    - motion_ids
+    """
+
+    collection = Collection("motion_change_recommendation")
+    verbose_name = "motion_change_recommendation"
+
+    id = fields.IdField(description="The id of this motion change recommendation.")
+    # Replacement (0), Insertion (1), Deletion (2), Other (3)
+    type = fields.IntegerField(
```
Comment (jsangmeister) [inline openslides_backend/models/motion_change_recommendation.py:19]: Just use a `PositiveIntegerField`

File: openslides_backend/models/motion_change_recommendation.py [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+from . import fields
+from .base import Model
+
+
+class MotionChangeRecommendation(Model):
+    """
+    Model for motion change_recommendation.
+
+    Reverse fields:
+    - motion_ids
```
Comment (jsangmeister) [inline openslides_backend/models/motion_change_recommendation.py:11]: remove

**Review by jsangmeister — APPROVED** [review]
minor changes, rest looks good!

File: openslides_backend/models/motion_change_recommendation.py (hunk 3 above)
Comment (normanjaeckel) [reply openslides_backend/models/motion_change_recommendation.py:38]: So this has to be set via the update_instance hook and should be removed here.

File: openslides_backend/models/motion_change_recommendation.py (hunk 4 above)
Comment (normanjaeckel) [reply openslides_backend/models/motion_change_recommendation.py:19]: Our PositiveIntegerField does not support zero. So you can use IntegerField with enum=[0,1,2,3],

Comment (normanjaeckel) [reply openslides_backend/models/motion_change_recommendation.py:19]: I think this has also a default value.

**Review by normanjaeckel — CHANGES_REQUESTED** [review]
See some minor remarks.

File: openslides_backend/models/motion_change_recommendation.py (hunk 3 above)
Comment (reiterl) [reply openslides_backend/models/motion_change_recommendation.py:38]: Done, set the current timestamp in update_instance and add tests for it.

File: openslides_backend/models/motion_change_recommendation.py (hunk 4 above)
Comment (reiterl) [reply openslides_backend/models/motion_change_recommendation.py:19]: Done. Use enum and set a default value to 0.

File: openslides_backend/models/motion_change_recommendation.py (hunk 5 above)
Comment (reiterl) [reply openslides_backend/models/motion_change_recommendation.py]: Done.

File: openslides_backend/models/motion_change_recommendation.py (hunk 4 above)
Comment (jsangmeister) [reply openslides_backend/models/motion_change_recommendation.py:16]: > Our PositiveIntegerField does not support zero.
Ah, my bad.

**Review by jsangmeister — CHANGES_REQUESTED** [review]
Adjust to new base test class functions. Rest is good

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**